src/Adapter/PlatformServices.Desktop/Utilities/FileUtility.cs

```diff
@@ -293,9 +293,15 @@ internal virtual string[] GetDirectoriesInADirectory(string directoryPath)
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes", Justification = "Requirement is to handle all kinds of user exceptions and message appropriately.")]
         private string GetSymbolsFileName(string path)
         {
+            Debug.Assert(!string.IsNullOrEmpty(path), "path");
+
             try
             {
-                return DiaHelper.GetSymbolsFileName(path);
+                string pdbFile = Path.ChangeExtension(path, ".pdb");
```

Comment:
relying on a catch to take care of checks isn't the right paradigm. The fxcop suppression on top of the method itself is a red flag. W can remove the catch below and pre-pone checks if you think this would not cause any unforeseen errors.